### PR TITLE
Manage: Add coin:privacy:dpa_type to SP metadata configuration

### PR DESCRIPTION
**Pull request by quartje**
This commit is part of the effort to refactor privacy related metadata.

### CODE REVIEW

File: roles/manage-server/files/metadata_configuration/oidc10_rp.schema.json.j2 [hunk 1]
```diff
@@ -412,6 +412,18 @@
...
+          "type": "string",
+          "enum": [
+            "service_is_not_processor",
+            "service_aqcuired_through_surf",
+            "service_willing_or_signed_surf_model_dpa",
+            "service_has_own_dpa",
+            "Other"
+           ],
+          "default" : "service_has_own_dpa",
+          "info": "Determines what typr of DPA this service has to offer"
```
Comment (thijskh) [inline roles/manage-server/files/metadata_configuration/oidc10_rp.schema.json.j2:425]:
```suggestion
          "info": "Determines what DPA this service has to offer"
```

File: roles/manage-server/files/metadata_configuration/oidc10_rp.schema.json.j2 [hunk 2]
```diff
@@ -412,6 +412,18 @@
...
           "info": "Explains why the SP answered no on the subject of the SURFmarket DPA."
         },
+        "coin:privacy:dpa_type": {
+          "type": "string",
+          "enum": [
+            "service_is_not_processor",
+            "service_aqcuired_through_surf",
+            "service_willing_or_signed_surf_model_dpa",
+            "service_has_own_dpa",
+            "Other"
```
Comment (thijskh) [inline roles/manage-server/files/metadata_configuration/oidc10_rp.schema.json.j2:422]:
```suggestion
            "other"
```

File: roles/manage-server/files/metadata_configuration/saml20_sp.schema.json.j2 [hunk 3]
```diff
@@ -510,6 +510,18 @@
...
           "info": "Explains why the SP answered no on the subject of the SURFmarket DPA."
         },
+        "coin:privacy:dpa_type": {
+          "type": "string",
+          "enum": [
+            "service_is_not_processor",
+            "service_aqcuired_through_surf",
+            "service_willing_or_signed_surf_model_dpa",
+            "service_has_own_dpa",
+            "Other"
```
Comment (thijskh) [inline roles/manage-server/files/metadata_configuration/saml20_sp.schema.json.j2:520]:
```suggestion
            "other"
```

File: roles/manage-server/files/metadata_configuration/oidc10_rp.schema.json.j2 [hunk 4]
```diff
@@ -412,6 +412,18 @@
           "type": "string",
           "info": "Explains why the SP answered no on the subject of the SURFmarket DPA."
         },
+        "coin:privacy:dpa_type": {
+          "type": "string",
+          "enum": [
+            "service_is_not_processor",
```
Comment (teunfransen) [inline roles/manage-server/files/metadata_configuration/oidc10_rp.schema.json.j2:418]: dpa_not_applicable

File: roles/manage-server/files/metadata_configuration/oidc10_rp.schema.json.j2 [hunk 5]
```diff
@@ -412,6 +412,18 @@
...
+            "service_aqcuired_through_surf",
```
Comment (teunfransen) [inline roles/manage-server/files/metadata_configuration/oidc10_rp.schema.json.j2:419]: dpa_in_surf_agreement

File: roles/manage-server/files/metadata_configuration/oidc10_rp.schema.json.j2 [hunk 6]
```diff
@@ -412,6 +412,18 @@
...
+            "service_willing_or_signed_surf_model_dpa",
```
Comment (teunfransen) [inline roles/manage-server/files/metadata_configuration/oidc10_rp.schema.json.j2:420]: dpa_model_surf

**Pull request merged**